_docs/_latest/configure/notifier/index.md

```diff
@@ -2,4 +2,144 @@
 title: Notifier
 order: 300
 ---
+
 # {{ page.title }}
+
+Forseti Notifier can send a variety of notifications to alert you
+of Forseti events. You can configure notifications to send to different
+channels and in different formats.
+
+### Notification Types
+
+  1. Inventory summary
+  1. Scanner summary
+  1. Violations
+  1. Cloud Security Command Center (Cloud SCC) findings
+  
+### Notification Channels
+
+  1. Email
+  1. Slack
+  1. Cloud Storage
+
+### Notification Formats
+
+  1. Human-readable data: .csv
+  1. Structured data: JSON
+
+## Configuring Notifier
+
+### Inventory Summary (TBD)
+
+### Scanner Summary (TBD)
+
+### Violation Notifications
+
+To select how you want Notifier to send violation notifications,
+follow the steps below:
+
+1. Open `forseti-security/configs/forseti_conf.yaml`.
+1. Navigate to the `notifier` > `resources` section.
+
+The following options are available, on a per resource basis. You can use
+any combination of notifiers for each resource.
+
+* `should_notify`: Controls whether violation for each resource should be sent.
+  `true` enables the notification, and `false` disables the notification.
+
+* `name`: The name of the notifier you want for each resource.  You can specify
+  multiple notifiers for each resource.
+
+  The name must match the actual module name for each notifier in 
+  `forseti-security/google/cloud/forseti/notifier/notifiers`,
```

Comment:
Is there a way to share (and keep up to date a list) of possible notifiers here?